ha/entrypoint.c

```diff
@@ -489,7 +489,7 @@ static void ha_entrypoint_client_replied(struct m0_rpc_item *item)
 	}
 	/* XXX TODO Handle rpc errors. */
 	m0_fom_wakeup(&ecl->ecl_fom);
-	m0_rpc_item_put(item); /* XXX Bug if rc != 0 ? */
+	m0_rpc_item_put(item);
```

Comment:
What if the `rc` is not zero?
There would be no matching m0_rpc_item_get() for this put.
Do you think we need:
```
if (rc == 0) {
        m0_rpc_item_put(item);
}
```

Reply:
@huanghua78, the above get() is for `item->ri_reply`, but this put() is for `item` - they are not matching in any case.